mithril.js

```diff
@@ -1959,9 +1988,17 @@
 		}
 	}
 
+	function isNativeError(e) {
```

Comment:
The current method works across frames. Why do you want to change it?

Part of the conversation got lost in the GitHub Web UI:

*On Wed, Mar 30, 2016 at 10:27 PM, Pierre-Yves Gérardy <pygy79@gmail.com> wrote:*
> *On Wed, Mar 30, 2016 at 9:34 PM, Isiah Meadows <notifications@github.com> wrote:*
>> @pygy This is for m.deferred.onerror(e).
>
> Do you expect all these errors to come strictly from the main `window`?